src/descriptor.rs

```diff
@@ -266,27 +269,57 @@ impl Descriptor {
         }
     }
 
-    /// Constructs a new [`Descriptor`] from a P2TR payload.
+    /// Constructs a new [`Descriptor`] from an *unckecked* P2TR payload.
```

Comment:
```suggestion
    /// Constructs a new [`Descriptor`] from an *unchecked* P2TR payload.
```

Reply:
Oh crap, done in 741b0d8fee73521abe59188d0eb88f95b04f74f5

Nice catch!